SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -42,6 +43,22 @@ FName USpatialStatics::GetCurrentWorkerType(const UObject* WorldContext)
 	return NAME_None;
 }
 
+bool USpatialStatics::GetWorkerFlag(const UObject* WorldContext, const FString& InString, FString& OutString)
```

Comment:
Can this function be a const function?

Reply:
No because it is a static function. I will revert the change.